app/scripts/modules/core/pipeline/config/stages/manualJudgment/ManualJudgmentApproval.tsx

```diff
@@ -61,51 +63,64 @@ export class ManualJudgmentApproval extends React.Component<IProps, IState> {
 
   private handleStopClick = () => this.provideJudgment('stop');
 
+  private getInstructions = (): any => {
+    return {
+      __html: DOMPurify.sanitize(this.props.stage.context.instructions)
+    };
+  };
```

Comment:
Unfortunately, it won't be the last time we have to do this. Probably worth refactoring into a common module next time we do it.